### PR TITLE
Black Hole Particle Effects and Parking Indicator

### PR DESCRIPTION
**Pull request by DJSteins**
- Adds a prefab that uses Unity particle effects for the black hole power-up
- Adds a reactive parking indicator

### CODE REVIEW

**Comment by DJSteins** [comment]
[Black hole and parking indicator](https://trello.com/c/xurUYRCZ/345-black-hole-and-parking-indicator)

File: Assets/Epic Toon FX/Materials/Portals/portal_stripes_soft.mat [hunk 1]
```diff
@@ -24,6 +24,6 @@ Material:
         m_Scale: {x: 1, y: 1}
         m_Offset: {x: 0, y: 0}
     m_Floats:
-    - _InvFade: 1
+    - _InvFade: 0.75
```
Comment (m4rtint) [inline Assets/Epic Toon FX/Materials/Portals/portal_stripes_soft.mat]: Just keep in mind that the materials are universal, so if you want to use the original one, you'll need to revert back to the original material properties

File: Assets/Epic Toon FX/Prefabs/Interactive/Powerups/PowerOrb/PowerOrbBlue.prefab [hunk 2]
```diff
@@ -5124,7 +5124,7 @@ ParticleSystem:
         m_RotationOrder: 4
     startRotationX:
       serializedVersion: 2
-      minMaxState: 0
+      minMaxState: 3
```
Comment (m4rtint) [inline Assets/Epic Toon FX/Prefabs/Interactive/Powerups/PowerOrb/PowerOrbBlue.prefab]: Did you make any changes to the powerorb blue?

Comment (DJSteins) [reply Assets/Epic Toon FX/Prefabs/Interactive/Powerups/PowerOrb/PowerOrbBlue.prefab]: Oh I may have accidentally done that when I was looking at different effects

Comment (m4rtint) [reply Assets/Epic Toon FX/Prefabs/Interactive/Powerups/PowerOrb/PowerOrbBlue.prefab]: yeah as long as there's nothing significant change, it's fine

File: Assets/Script/PowerUp/GravityWellBehaviour.cs [hunk 3]
```diff
@@ -100,4 +100,10 @@ private void OnTriggerExit(Collider other)
             m_lengthOfCarsList--;
         }
     }
+
+    
+    protected override float GetYCoord()
+    {
+        return 0.25f;
```
Comment (Gursaanj) [inline Assets/Script/PowerUp/GravityWellBehaviour.cs]: why do you want such a specific float to be returned here?

Comment (DJSteins) [reply Assets/Script/PowerUp/GravityWellBehaviour.cs]: It just works to put the indicator right above the ground so it doesn't clip below the map

**Pull request merged**